### ISSUE TITLE
Can't start broadcasting

### ISSUE THREAD

**Issue by PawanmSpaceo** [SKrotkih/YTLiveStreaming#7]
while starting broadcast getting error of "Something went wrong with creating a broadcast"

**Comment by Jignesh-Gangajaliya** [comment]
Getting Error While Starting Broadcast. it was working fine before.

System Error: Response status code was unacceptable: 403.
Something went wrong with creating a broadcast.

@SKrotkih  Can you please resolve ASAP as I need to show a demo to the client.

Thanks.

**Comment by PawanmSpaceo** [comment]
@JackyGajjar do we need 1k subscription to start broadcasting?

**Comment by Jignesh-Gangajaliya** [comment]
@PawanmSpaceo  yep, Youtube imposed new restrictions on users: "To be able to live stream on mobile, your channel will need to have at least 1,000 subscribers.".
I have tried on my Channel using YTLiveStreaming it was working but suddenly getting an error.
I haven't 1k subscription.

**Comment by PawanmSpaceo** [comment]
Okay @JackyGajjar

**Comment by Jignesh-Gangajaliya** [comment]
> Getting Error While Starting Broadcast. it was working fine before.
> 
> System Error: Response status code was unacceptable: 403.
> Something went wrong with creating a broadcast.
> 
> @SKrotkih Can you please resolve ASAP as I need to show a demo to the client.
> 
> Thanks.

@SKrotkih  any update?

**Comment by SKrotkih** [comment]
Thanks for the report, and apologies for the long silence — the project was dormant for a while. It is being revived as **1.0**, a from-scratch rewrite (URLSession + async/await, no third-party dependencies, SPM only): https://github.com/SKrotkih/YTLiveStreaming/tree/feature/v1-urlsession

"Something went wrong with creating a broadcast" was a generic message that hid the real cause (usually: live streaming not enabled on the channel, missing OAuth scopes, or quota). 1.0 surfaces Google's actual error payload — `YouTubeLiveError.forbidden(GoogleAPIError)` with `reason` such as `liveStreamingNotEnabled` or `insufficientPermissions` — so the failure is diagnosable. Closing; please reopen with the `reason` string if you hit it on 1.0.
